templates/angular/ts/hierarchical-grid/hierarchical-grid/files/src/app/components/__path__/__name__.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component } from "@angular/core";
+import { northwind } from "../../assets/northwind";
+
+@Component({
+	selector: "app-grid",
```

Comment:
Is this selector used for the route navigation? If so we need to update it to app-hierarchical-grid?